docs/index.md

```diff
@@ -35,6 +35,57 @@ non-public development, custom hardware handled by vendor
   source hardware and open instruction set architecture. Feel free to check the
   website about recent vPub plans or subscribe to newsletter.
 
+## Newcommers
+
+If you are new to Dasharo, this section it to help you get started using Dasharo
+firmware and possibly contributing to making it better.
+
+### Coreboot
+
+If you have no prior experience with coreboot, it's highly recommended to visit
+OpenSecurityTraining2 and finish these courses:
+* [Arch4031](https://p.ost2.fyi/courses/course-v1:OpenSecurityTraining2+Arch4031_x86-64_RV_coreboot+2021_v1/about),
+* [Arch4021](https://p.ost2.fyi/courses/course-v1:OpenSecurityTraining2+4021_Intro_UEFI+2022_v1/about)
+OpenSecurityTraining2 provides other great courses related to firmware, which
+are all freely avialable [here](https://p.ost2.fyi/courses).
+
+### Docker
+
+Coreboot needs a specific toolchain to be built, hence why it is usually built
+inside of a Docker. In case of Dasharo, all images are build using Docker, so
+you will make sure it works properly on your system.
+
+Follow these two links:
+* [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+* [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
```

Comment:
@Stojek139808 IMHO this is not enough we have to point to documentation how to use coreboot-sdk, if there is no such documentation it has to be created - I also explain what you wrote here in Arch4031. To some extent it breaks DRY rule.

Reply:
It does a bit, but at the same time ost2 requires an account to acces, i feel like this should be accessible here, also arch4031 is already linked above.
Other than that, I've added the typical way to build coreboot with docker. Also I don't understad why this is not described in docs.coreboot, I remember the lack of build process documentation was very confusing to me when I was starting out. Maybe this is something we could contribute in upstream.

Reply:
> It does a bit, but at the same time ost2 requires an account to acces, i feel like this should be accessible here, also arch4031 is already linked above.

Account can be created with fake email, but you are right it doesn't make sense to create ost2 account to get knowledge about how to compile Dasharo. We need that information in Dasharo documentation and we have it for supported hardware, maybe when we add QEMU as target then it would be canonical learning path for Dasharo newcomers.

> Other than that, I've added the typical way to build coreboot with docker. Also I don't understad why this is not described in docs.coreboot, I remember the lack of build process documentation was very confusing to me when I was starting out. Maybe this is something we could contribute in upstream.

Of course, but I afraid not everyone support building in containers as well such process may be quickly outdated and lead to lot of inquiries. We can try to contribute such documentation. I hope it can get through. And I'm completely with you open-source projects need clear information about building environment, it cannot be obstacle it should work without any issues.